### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by hamza1311**
This PR should be created automatically but I don't permissions to allow GitHub actions to do that. Right now, it fails with:
```
pull request create failed: GraphQL: Resource not accessible by integration (createPullRequest)
```
See [logs](https://github.com/rustwasm/wasm-bindgen/actions/runs/6706407515/job/18222833380#step:12:9)

@alexcrichton can you [allow actions to create PRs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#preventing-github-actions-from-creating-or-approving-pull-requests).  
Please also publish this release

Fixes: #3530

### CODE REVIEW

**Comment by hamza1311** [comment]
@alexcrichton yes, please do add the secret. I would also like to reduce it down to one step, so adding a secret is necessary. You can go ahead and publish this release. We can automate the next one

**Comment by alexcrichton** [comment]
Ok! I've added `CRATES_PUBLISH_TOKEN` as a crate secret here. It's scoped to `wasm-bindgen*`, `js-sys`, and `web-sys` only, but I think that should work?

Otherwise I'll defer to y'all when to merge this, but when merged I'll tag this and publish it.

**Comment by Liamolucko** [comment]
There's one more version that still needs to be bumped, which is the 'Unreleased' heading in `CHANGELOG.md` that should now become 0.2.88. That should be it though, looking at #3475 for a reference.

**Comment by hamza1311** [comment]
I bumped it manually this time but we should update the script to automate this

File: CHANGELOG.md
Comment (Liamolucko) [inline CHANGELOG.md:6]:
```suggestion
Released 2023-11-01
```

**Review by daxpedda — APPROVED** [review]

**Comment by hamza1311** [comment]
I'm going to merge this for @alexcrichton to publish and tag. We can use the automated workflow from next release (I would love to fix it now but unfortunately I got ill...)

**Comment by alexcrichton** [comment]
Ok published and tagged 👍

**Pull request merged**
